src/lib.rs

```diff
@@ -297,6 +298,7 @@ pub use self::decode_all::DecodeAll;
 pub use self::depth_limit::DecodeLimit;
 pub use self::encode_append::EncodeAppend;
 pub use self::encode_like::{EncodeLike, Ref};
+#[cfg(feature = "max-encoded-len")]
```

Comment:
You also probably need to add this to `pub use parity_scale_codec_derive::MaxEncodedLen` below.

Reply:
Done in https://github.com/paritytech/parity-scale-codec/pull/282/commits/5c022a4250244afd0294b6963640184884ac3cbd

Reply:
Thanks for spotting it!